### PR TITLE
Css three column layout

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-01-27 at 7 58 04 PM

### CODE REVIEW

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-28 at 1 17 52 PM

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.row {
+  display: flex;
+}
+
+.column-main {
+  flex-basis: 100%;
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: Column classes should only include `flex-basis` or `width`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+.paddingtop {
+  padding-top: 8px;
+}
+
+.column {
+  flex-basis: calc(100% / 3);
+  align-content: center;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: Column classes should only include `flex-basis` or `width`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-28 at 5 02 47 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jacob, the font size and weight for the subheader and text in each column do not match the example, please revise and submit a new screenshot with your revision. It may be helpful to use the actual text so that you can gauge the quality of your output better.

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-31 at 3 04 58 PM

File: css-three-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+
+    <div class="row">
+      <div class="column-main">
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html]: `column-main`? What's its size? This isn't how `column` classes are meant to be named.

File: css-three-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <h2 class="paddingtop textalign">Lorem ipsum, dolor sit amet consectetur adipisicing elit.</h2>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column">
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html]: `column` what?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-02-01 at 10 56 59 AM
thank you, i changed the names of the coulmns

**Pull request merged**